pkg/log/logger.go

```diff
@@ -106,6 +106,11 @@ func (l logger) Errorf(format string, args ...interface{}) {
 var origLogger = logrus.New()
 var baseLogger = logger{entry: logrus.NewEntry(origLogger)}
 
+// Orig provides access to the underlying *logrus.Logger
```

Comment:
I added this to try and get the rethinkdb driver logging but it hasn't worked quite yet